core/src/autogluon/core/models/ensemble/bagged_ensemble_model.py

```diff
@@ -28,7 +25,7 @@ class BaggedEnsembleModel(AbstractModel):
     """
     _oof_filename = 'oof.pkl'
 
-    def __init__(self, model_base: AbstractModel, random_state=0, **kwargs):
+    def __init__(self, model_base: AbstractModel, fold_fitting_strategy: AbstractFoldFittingStrategy = None, random_state=0, **kwargs):
```

Comment:
fold_fitting_strategy should be a hyperparameter, not a parameter listed here

Reply:
passed via ag_args_ensemble instead